wcivf/apps/peoplecvs/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from people.models import Person
+
+
+class CV(models.Model):
+    """
+    A candidate's CV.
+    """
+    person = models.ForeignKey(Person)
```

Comment:
no, it's definitely one-to-one (checked with Francis) - so yes this should be a `OneToOneField`